tests/conftest.py

```diff
@@ -91,7 +91,8 @@ def sigterm_handler(_signo: int, _stack_frame: Any) -> None:
         while not status == 200 and count < max_tries:
             try:
                 count += 1
-                r = requests.get(f"http://{self.web_host}:{self.web_port}/health")
+                r = requests.get(f"http://{self.web_host}:{self.web_port}/health",
```

Comment:
I ran into this problem too, do you know why the header became mandatory?

Reply:
I'm sure it was a package update somewhere. Could potentially be handled better in the server